sasl/external/src/test/java/org/wildfly/security/sasl/external/ExternalSaslClientTest.java

```diff
@@ -213,13 +214,19 @@ public void testServerChallengeEmptyAuthzId() throws Exception {
     /**
      * Test failing (as we only authenticate "admin") authn for unsupported data "test" from client.
      */
-    @Test(expected = SaslException.class)
+    @Test
     public void testWrongServerChallenge() throws Exception {
```

Comment:
@keshav-redhat Just a minor, we can remove throws Exception from the method's declaration now, because you have added the try catch block